sos/report/plugins/openvswitch.py

```diff
@@ -147,18 +152,16 @@ def setup(self):
                             "ovs-ofctl -O %s dump-ports-desc %s" % (flow, br)
                         ])
 
-        # Gather info on the DPDK mempools associated with each DPDK port
-        br_list_result = self.collect_cmd_output("ovs-vsctl -t 5 list-br")
-        if br_list_result['status'] == 0:
-            for br in br_list_result['output'].splitlines():
                 port_list_result = self.exec_cmd(
                     "ovs-vsctl -t 5 list-ports %s" % br
                 )
                 if port_list_result['status'] == 0:
                     for port in port_list_result['output'].splitlines():
-                        self.add_cmd_output(
-                            "ovs-appctl netdev-dpdk/get-mempool-info %s" % port
-                        )
+                        if check_dpdk:
+                            self.add_cmd_output(
+                                "ovs-appctl netdev-dpdk/get-mempool-info %s" %
```

Comment:
The mempool info is missing when I ran with dpdk-init=true and 1x dpdk and 1x dpdkvhostclient ports

Reply:
# ls ovsmaster/*mempool*
ovsmaster/ovs-appctl_netdev-dpdk.get-mempool-info_dpdkvhost0  ovsmaster/ovs-appctl_netdev-dpdk.get-mempool-info_myport

# ls ovsaaron/*mempool*
ls: cannot access 'ovsaaron/*mempool*': No such file or directory

# ovs-vsctl -t 5 get Open_vSwitch . other_config:dpdk-init
"true"

# ovs-appctl netdev-dpdk/get-mempool-info dpdkvhost0 | grep mempool
mempool <ovs530eb21900021580262144>@0x23f7707c0

Reply:
Yep - the test is incorrect.  A correct version needs to account for " being included.